webpack.config.ts

```diff
@@ -291,6 +311,12 @@ module.exports = {
     new HtmlWebpackPlugin({
       chunks: ['library'],
       filename: 'library-page.mainpage.html',
+      meta: {
+        name: 'Personalized Online Learning from Oppia',
+        description: 'Looking to learn something new? Find explorations ' +
```

Comment:
Will it work as before? (different description for search page and library main page)